README.md

```diff
@@ -20,46 +20,27 @@ Go to https://azure.microsoft.com/en-us/services/app-service/ to create a Free A
+| SLACK_VERIFICATION_TOKEN     | A value from the Basic Information -> Verification Token field in the settings for your slack app.                                                                                                                                                                                                                                                                                                                                                                                           |
+| SLACK_SIGNING_SECRET         | Secret from the App Credentials page for your app in Slack.                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| POST_TO_CHANNEL              | a boolean value `True` or `False` that indicates whether or not to take the modal data and post to a channel in slack                                                                                                                                                                                                                                                                                                                                                                        |
+| CHANNEL                      | Tthe channel id (such as C01DB7S04KH -> NOT THE NAME) you want the modal results to post to by default. other values supported. set to `THE_AO` to post to the channel that was selected in the modal by default. Set to `USER` to post a DM from the slackblast to you with the results (testing) by default. If blank or missing, then the default channel will be the channel the user typed the slash command. In the modal, the user can choose the "destination" and where to post to. |
+| EMAIL_SERVER                 | SMTP Server to use to send the email                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
```

Comment:
Recommend updating this to help gmail server users. I would suggest something along these lines.

`SMPT Server to use to send the email, default is "smtp.gmail.com" so if sending from a gmail account you only need to fill out email_user and email_password`